code/modules/sanity/sanity_mob.dm

```diff
@@ -89,6 +93,9 @@
 	owner = H
 	level = max_level
 	insight = rand(0, 30)
+	insight_passive_gain_multiplier = 0.5 * GLOB.GLOBAL_INSIGHT_MOD //set them here so mod variables could initialize
+	insight_gain_multiplier = 1 * GLOB.GLOBAL_INSIGHT_MOD
+	insight_rest_gain_multiplier = 1 * GLOB.GLOBAL_INSIGHT_MOD
```

Comment:
Bruh, put it into actual sanity calculations, that only would only proc on mob creation.

Reply:
(fixed this on local, will update after parallax torture is finished)